src/pykeen/evaluation/evaluator.py

```diff
@@ -35,10 +35,17 @@
     "MetricResults",
     "filter_scores_",
     "evaluate",
+    "SIDE_HEAD",
+    "SIDE_RELATION",
+    "SIDE_TAIL",
 ]
 
 logger = logging.getLogger(__name__)
 
+SIDE_HEAD = "head"
+SIDE_TAIL = "tail"
+SIDE_RELATION = "relation"
```

Comment:
move to constants? I am sure there are other uses of these constants in pykeen

Reply:
Done in 88cad831